app/controllers/catalog_controller.rb

```diff
@@ -113,7 +113,8 @@ class CatalogController < ApplicationController
     # solr fields to be displayed in the index (search results) view
     #   The ordering of the field names is the order of the display
     config.add_index_field 'title_tesi', label: 'Title', if: Proc.new {|context, _field_config, _document| context.request.format == :json }
-    config.add_index_field 'date_issued_ssi', label: 'Date', helper_method: :combined_display_date
+    config.add_index_field 'date_issued_ssi', label: 'Publication date'
+    config.add_index_field 'date_created_ssi', label: 'Creation date'
```

Comment:
Good catch!  I forgot about the combined dates on search results.  This probably also comes up in collection landing page search results.

@joncameron Which dates do you think should be displayed on search results?

Reply:
Could it be done in a way that favors Publication Date, but shows Creation Date if it's present and Publication Date is not? 

Something like:
```ruby
if date_created.present? && !date_issued.present?
    config.add_index_field 'date_created_ssi', label: 'Creation date'
else
    config.add_index_field 'date_issued_ssi', label: 'Publication date'
end
```

Reply:
That might be possible.  We'd have to use a helper_method(s) since the config is static and conditionals can't be evaluated here.

Reply:
+1 to adding a helper method like the previous used combined_date if that's reasonable. The label could even continue to be generic "Date" if that makes things easier.